.github/workflows/deploy-stack.yml

```diff
@@ -76,12 +77,27 @@ jobs:
             echo 'services=["${{ inputs.service }}"]' >> "$GITHUB_OUTPUT"
           fi
 
+  notify-start:
+    name: Start Deployment Notification
+    runs-on: ubuntu-latest
+    if: github.event.inputs.environment == 'production'
```

Comment:
I _think_ you need to use `${{ inputs.environment }}` since this uses both workflow call and dispatch. I seem to remember getting caught up on this at one point.